frontend/src/features/leases/detail/LeasePages/deposits/components/DepositsReceivedContainer/columns.tsx

```diff
@@ -29,14 +29,24 @@ export class DepositListEntry {
     this.depositDescription = baseDeposit.description;
     this.amountPaid = baseDeposit.amountPaid;
     this.paidDate = baseDeposit.depositDate || '';
-    this.personDepositHolder = baseDeposit.personDepositHolder;
-    this.organizationDepositHolder = baseDeposit.organizationDepositHolder;
+    this.contactHolder = baseDeposit.contactHolder;
     this.depositReturnCount = baseDeposit.depositReturns.length;
   }
 }
 
 function renderHolder({ row: { original } }: CellProps<DepositListEntry, string>) {
-  return original.personDepositHolder?.fullName || '';
+  if (original.contactHolder !== undefined) {
+    const holder = original.contactHolder;
+    if (holder.person !== undefined) {
+      return (
+        holder.person.firstName + ' ' + holder.person.middleNames + ' ' + holder.person.surname
```

Comment:
I've seen a few places in this PR where a full name is formatted. Please consider using a shared utils functions (already existing) - `formatFullName` in **utils/personUtils.ts**

Also this specific line is a bit buggy in that if any of the person fields above is undefined you would get the following:

```
holder.person.firstName + ' ' + holder.person.middleNames + ' ' + holder.person.surname == "First undefined Last"
```

Because undefined casted to string yields the string "undefined"

Reply:
All the more reason to use a shared utility